contracts/v6/interface/IUniswapV2Factory.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity =0.6.6;
+pragma solidity ^0.6.6;
```

Comment:
Keep these an absolute reference to a version... so =0.6.12. This was based on feedback from Certik.

Reply:
Done

Reply:
This is not done

Reply:
Fixed